--- origsrc/libarchive-3.2.2/libarchive/archive_write_disk_posix.c	2016-09-11 16:28:18.000000000 -0500
+++ src/libarchive-3.2.2/libarchive/archive_write_disk_posix.c	2016-11-10 13:11:52.125558700 -0600
@@ -2615,7 +2615,7 @@ check_symlinks(struct archive_write_disk
 }
 
 
-#if defined(__CYGWIN__)
+#if 0 // defined(__CYGWIN__)
 /*
  * 1. Convert a path separator from '\' to '/' .
  *    We shouldn't check multibyte character directly because some
@@ -2702,9 +2702,6 @@ cleanup_pathname_fsobj(char *path, int *
 		return (ARCHIVE_FAILED);
 	}
 
-#if defined(__CYGWIN__)
-	cleanup_pathname_win(a);
-#endif
 	/* Skip leading '/'. */
 	if (*src == '/') {
 		if (flags & ARCHIVE_EXTRACT_SECURE_NOABSOLUTEPATHS) {
