--- origsrc/libarchive-2.4.17/Makefile.am	2008-03-29 13:23:17.000000000 -0500
+++ src/libarchive-2.4.17/Makefile.am	2008-05-28 07:35:25.453125000 -0500
@@ -136,7 +136,7 @@
 
 # archive.h ends up in the build directory, so make sure we can find it.
 libarchive_la_CPPFLAGS= -I$(top_builddir)/libarchive
-libarchive_la_LDFLAGS= -version-info $(ARCHIVE_LIBTOOL_VERSION)
+libarchive_la_LDFLAGS= -version-info $(ARCHIVE_LIBTOOL_VERSION) -no-undefined
 
 # Manpages to install
 libarchive_man_MANS=						\
@@ -223,7 +223,7 @@
 	libarchive/test/test_write_format_tar_empty.c		\
 	libarchive/test/test_write_open_memory.c
 
-libarchive_test_CPPFLAGS= -I$(top_srcdir)/libarchive -I$(top_builddir)/libarchive/test
+libarchive_test_CPPFLAGS= -I$(top_srcdir)/libarchive -I$(top_builddir)/libarchive -I$(top_builddir)/libarchive/test
 
 # The "list.h" file just lists all of the tests defined in all of the sources.
 # Building it automatically provides a sanity-check on libarchive_test_SOURCES
